scripts/install.ps1

```diff
@@ -76,7 +76,7 @@ If ($IsLinux) {
 # Pillow, pygetwindow, pymsgbox, pytweening, MouseInfo are picked up by PySide6
 # (also --exclude from build script, but more consistent with unfrozen run)
 &"$python" -m pip uninstall pyscreenshot mss pygetwindow pymsgbox pytweening MouseInfo -y
-If ($IsWindows) { &"$python" -m pip uninstall pyscreeze Pillow -y }
```

Comment:
I assume this is for pytesseract ? Does it not have an OpenCV implementation ?
There's nothing wrong with Pillow by itself, just that it might be a redundant dependency (and I just got rid of it on Windows! #266 ). And we're also using OpenCV already, which is usually more performant due to being a C-extension.

Reply:
Yes you are right. There is a OpenCV interface for pytesseract.

Reply:
@Avasam 
Well, nevermind.

I tested pretty much all of the python OCR libraries and it seems that all of them import Pillow somewhere in their codebase.

I even tried to mock it (PIL.py):
```
class Image:
    Image = None
    def __init__(self):
        yield
class ImageColor:
    def __init__(self):
        yield
class ImageDraw:
    def __init__(self):
        yield
class ImageFont:
    def __init__(self):
        yield
```

But this does not work for imports like this: `import PIL.Image`

So I understand your incentive getting rid of Pillow but since it is used somewhere in all of these libs there seems no other option.
Do you have another idea?

Reply:
Pillow as a dependency isn't bad.
It's just a general principle of "need not, want not" (see my other comment about the current EasyOCR situation)